examples/roads2png.rs

```diff
@@ -233,41 +195,49 @@ fn render(archive: &osmflat::Osm, width: u32) -> Image {
     // draw
     let mut image = Image::new(width, height);
 
-    let lines = roads.flat_map(|(way, next_way)| {
-        let raster_coords = NodesIterator::from_way(archive, &way, &next_way)
-            .map(GeoCoord::from)
-            .map(|coord| t.transform(coord));
-        raster_coords.clone().zip(raster_coords.skip(1))
+    let lines = roads(archive).flat_map(|(way, next_way)| {
+        way_coords(archive, &way, &next_way)
+            .map(|coord| t.transform(coord))
+            .tuple_windows()
     });
 
     for (from, to) in lines {
         for (x, y) in Bresenham::new(from, to) {
-            image.set(x as u32, y as u32, Color::new(0, 0, 0, 255));
+            image.set_black(x as u32, y as u32);
         }
     }
 
     image
 }
 
+#[derive(StructOpt, Debug)]
+struct Opt {
+    #[structopt(short = "i", long = "input", parse(from_os_str))]
```

Comment:
I think I would remove short/long, then this becomes just a position argument and we would have something like gcc:
```
./bin input.file -o output.file
```

Reply:
Done